app/services/twilio_service.rb

```diff
@@ -25,7 +25,7 @@ def place_call(params = {})
 
   def send_sms(params = {})
     sanitize_errors do
-      params = params.reverse_merge(from: from_number)
```

Comment:
Remove these methods and constants as well?
 ```
from_number
account
random_account
config/initializers/twilio_account.rb
```

Reply:
We still need all of those to use those for voice calls. I would like to come in after this PR and remove the random account logic, since we just have one account, but that felt out of scope here.